data/presets/fields/surveillance/type.json

```diff
@@ -0,0 +1,8 @@
+{
+    "key": "surveillance:type",
```

Comment:
Ok, I see.  Yes I have heard of Surveillance Under Surveillance.  I didn't realize they were consuming this data and required `surveillance:type` to be filled in.  In that case, what you have for this file is fine 👍